code/modules/projectiles/targeting.dm

```diff
@@ -24,14 +24,14 @@
 //Removing the lock and the buttons.
 /obj/item/weapon/gun/dropped(mob/user as mob)
 	stop_aim()
-	if (user.client)
+	if (user && user.client)
```

Comment:
This is the precise purpose of `?.`
This works just as well though so I'm not going to make you change it